locales/nb/validation-inline.php

```diff
@@ -1,131 +1,143 @@
-|
-| The following language lines contain the default error messages used by
-| the validator class. Some of these rules have multiple versions such
-| as the size rules. Feel free to tweak each of these messages here.
-|
-*/
-
 return [
-    'accepted'             => 'This field must be accepted.',
-    'active_url'           => 'This is not a valid URL.',
-    'after'                => 'This must be a date after :date.',
-    'after_or_equal'       => 'This must be a date after or equal to :date.',
-    'alpha'                => 'This field may only contain letters.',
-    'alpha_dash'           => 'This field may only contain letters, numbers, dashes and underscores.',
-    'alpha_num'            => 'This field may only contain letters and numbers.',
-    'array'                => 'This field must be an array.',
-    'attached'             => 'This field is already attached.',
-    'before'               => 'This must be a date before :date.',
-    'before_or_equal'      => 'This must be a date before or equal to :date.',
+    /*
+    |--------------------------------------------------------------------------
+    | Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    'accepted'             => ':attribute må aksepteres.',
+    'active_url'           => ':attribute er ikke en gyldig URL.',
+    'after'                => ':attribute må være en dato etter :date.',
+    'after_or_equal'       => ':attribute må være en dato etter eller lik :date.',
+    'alpha'                => ':attribute må kun bestå av bokstaver.',
+    'alpha_dash'           => ':attribute må kun bestå av bokstaver, tall og bindestreker.',
+    'alpha_num'            => ':attribute må kun bestå av bokstaver og tall.',
+    'array'                => ':attribute må være en matrise.',
+    'attached'             => ':attribute er allerede vedlagt.',
+    'before'               => ':attribute må være en dato før :date.',
+    'before_or_equal'      => ':attribute må være en dato før eller lik :date.',
     'between'              => [
-        'array'   => 'This content must have between :min and :max items.',
-        'file'    => 'This file must be between :min and :max kilobytes.',
-        'numeric' => 'This value must be between :min and :max.',
-        'string'  => 'This string must be between :min and :max characters.',
+        'numeric' => ':attribute må være mellom :min - :max.',
+        'file'    => ':attribute må være mellom :min - :max kilobytes.',
+        'string'  => ':attribute må være mellom :min - :max tegn.',
+        'array'   => ':attribute må ha mellom :min - :max elementer.',
```

Comment:
sort

Reply:
Same here, compared to and changed according to the English file.